src/main/java/step3/LottoResult.java

```diff
@@ -0,0 +1,47 @@
+package step3;
+
+import step3.domain.Lotto;
+import step3.domain.LottoNumber;
+import step3.domain.LottoStatistic;
+import step3.domain.LottoWinning;
+import step3.view.InputView;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+public class LottoResult {
+	private final List<Lotto> issueLottos;
+	private LottoStatistic statistic;
+
+	private static final int LOTTO_PRICE = 1000;
+	private static final String DUPLICATE_BONUS_NUMBER_PHRASES = "로또 보너스 번호는 당첨 번호와 중복될 수 없습니다.";
+	private static final String BONUS_NUMBER_RANGE_PHRASES = "로또 보너스 번호는 1~45 중 한 개의 숫자로 이루어져야 합니다.";
```

Comment:
```suggestion
	private static final int LOTTO_PRICE = 1000;
	private static final String DUPLICATE_BONUS_NUMBER_PHRASES = "로또 보너스 번호는 당첨 번호와 중복될 수 없습니다.";
	private static final String BONUS_NUMBER_RANGE_PHRASES = "로또 보너스 번호는 1~45 중 한 개의 숫자로 이루어져야 합니다.";
	
	private final List<Lotto> issueLottos;
	private LottoStatistic statistic;
```
일반적으로 선언 순서는 상수 - 인스턴스 변수 순으로 선언합니다.